packages/manager/modules/web-paas/src/web-paas.service.js

```diff
@@ -189,6 +180,18 @@ export default class WebPaasService {
     );
   }
 
+  inviteUser(projectId, accountName) {
+    return this.$http.post(`/webPaaS/subscription/${projectId}/customer`, {
+      accountName,
+    });
+  }
+
+  deleteUser(projectId, customerId) {
+    return this.$http.delete(
+      `/webPaaS/subscription/${projectId}/customer/${customerId}`,
+    );
+  }
+
```

Comment:
Hello @varun257, I see this logic as part of the commit "[...] add tracking". Is this related to a feature ? Should it be done in another PR/commit ?

Reply:
This also included fixes for some qa issues @JeremyDec, i have added it in description of the commit